components/stepper/steps/validation.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Validation
+page_title: Validation
+description: Validation for the Stepper for Blazor.
+slug: stepper-steps-validation
+tags: telerik,blazor,stepper,steps,validation
+published: True
+position: 7
+---
+
+# Steps Validation
+
+The Stepper component allows you to set validation logic for each step. You can configure it through the `Valid` parameter of the `StepperStep` which accepts `bool?`.
```

Comment:
Consider adding "visual indication" somewhere here to clearly denote that this is just a visual for the end user and does not prevent them from navigating between steps. You can add it as its entire own sentence, of course.

Reply:
Done